draft-ietf-tls-dtls-rrc.md

```diff
@@ -142,15 +152,19 @@ cookie is a 8-byte field containing arbitrary data.
 The `return_routability_check` message MUST be authenticated and encrypted using
 the currently active security context.
 
-The receiver that observes the peer's address and or port update MUST stop
-sending any buffered application data (or limit the data sent to a TBD
-threshold) and initiate the return routability check that proceeds as follows:
+# Path Validation Procedure
 
-1. A cookie is placed in a `return_routability_check` message of type
-   path_challenge;
-1. The message is sent to the observed new address and a timeout T is started;
+The receiver that observes the peer's address or port update MUST stop sending
+any buffered application data (or limit the data sent to the unvalidated
+address to the anti-amplification limit) and initiate the return routability
+check that proceeds as follows:
+
+1. An unpredictable cookie is placed in a `return_routability_check` message of
+   type path_challenge;
+1. The message is sent to the observed new address and a timer T (see
+   {{timer-choice}}) is started;
 1. The peer endpoint, after successfully verifying the received
-   `return_routability_check` message echoes the cookie value in a
+   `return_routability_check` message responds by echoing the cookie value in a
    `return_routability_check` message of type path_response;
 1. When the initiator receives and verifies the `return_routability_check`
    message contains the sent cookie, it updates the peer address binding;
```

Comment:
1. What happens when the timer expires? 

Add something like: _"If the timer expires without receiving a path_response message, the procedure is stopped."_

1. What happens when the initiator receives an invalid path_response message?

This would be silently discarded.  (There is no impact on the timer logics.)

Reply:
> 1. What happens when the timer expires?
> 
> Add something like: _"If the timer expires without receiving a path_response message, the procedure is stopped."_

This was already there (hidden by GitHub's UI).

Reply:
addressed in df82fd8